containers/doc/server-kubernetes/rhn-ssl-tool.yaml

```diff
@@ -0,0 +1,54 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  labels:
+    run: rhn-ssl-tool
+  name: rhn-ssl-tool
+spec:
+  backoffLimit: 0
+  template:
+    spec:
+      restartPolicy: Never
+      initContainers:
+      - name: gen-ca
+        image: registry.opensuse.org/systemsmanagement/uyuni/master/servercontainer/containers/uyuni/server
+        command:
+        - rhn-ssl-tool
+        - --gen-ca
+        - --no-rpm
+        - --set-common-name=uyuni-dev.world-co.com
+        - --set-country=FR
+        - --set-state=Burgundy
+        - --set-city=Macon
+        - --set-org=SUSE
+        - --set-org-unit=BCL
+        - --set-email=sylvestre@world-co.com
```

Comment:
not sure about this. do we keep FR as a cameo appearance of the author or do you plan to use the usual NUE info?

Reply:
removed the file